### PR TITLE
Fix/#78092 www.hltv.org

### PR DESCRIPTION
**Pull request by stanislav-atr**
#78092

### CODE REVIEW

**Review by wout-codes — COMMENTED** [review]
These rules do nothing on my end.

**Review by wout-codes — COMMENTED** [review]
`hltv.org##div[class*="ASdvuasdr123Gazx"]` This is (`ASdvuasdr123Gazx`) likely to change.

**Review by wout-codes — COMMENTED** [review]
There is also a popup window which you have missed.

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]
Please investigate this site again
Only these rules look necessary:

```
hltv.org##.pari-match
hltv.org##div[class*="ASdvuasdr123Gazx"]
||intersc.igaming-service.io/hltv.org.js
||hltv.org/scripts/hltv-tracking.js
hltv.org##.newMatches > div.gtSmartphone-only
```

And this site is not russian, so rules must be added to the base filter

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -112,6 +112,15 @@ mmo13.ru###wallpaper_container
...
 odessa-life.od.ua##div[class^="code-block code-block-"][style*="margin: 8px auto; text-align: center;"]
 vechirniy.kyiv.ua##.page-promo
+hltv.org##a[href^="https://betahit.click/ggbetting/"]
+hltv.org##a[href^="https://kinguin.deals/"][target="_blank"]
+hltv.org##a[href^="https://bitskins.com/hltv"][target="_blank"]
+hltv.org##div[class*="ASdvuasdr123Gazx"]
+hltv.org##.pari-match
+hltv.org##.gg-firstcol-box
+hltv.org###g4_bottom
+hltv.org##.footer-app
```
Comment (Sergey-Lyapin) [inline RussianFilter/sections/specific.txt:122]: We should not remove links to mobile apps in the footer.

**Comment by wout-codes** [comment]
@Sergey-Lyapin @stanislav-atr Most of the ads were fixed in EasyList. Only a `GGBet` banner is on the top of the page. So I recommend to changes these rules.

```
||intersc.igaming-service.io/hltv.org.js
||hltv.org/scripts/hltv-tracking.js
hltv.org#%#//scriptlet("remove-attr", "data-href", "body[class]")
hltv.org##.featured-matches-top
```

**Comment by zloyden** [comment]
@stanislav-atr also check old rules for this site.

**Review by zloyden — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**